pkg/proxy/proxy_test.go

```diff
@@ -92,6 +99,24 @@ func TestListNamespacedRelease(t *testing.T) {
 	}
 }
 
+func TestListOldRelease(t *testing.T) {
```

Comment:
Maybe extend this test or create another one that tests that the grouping happens by releaseName?

Reply:
if I understood your request, that's done in this test. It checks that the returned list only contain one element that is the newest version of a release (so it's being grouped by release name). I'll try to make it more clear though.

Reply:
Now I do not know how to see the old code :(

If I remember correctly, the test had a sample of two elements with the same release name and you were checking to receive a single one. This could mean that the function groups by name plus orders by version or just orders by version. My suggestion was about adding another release with a different releaseName with a higher version to double check that the per-name group + sorting happens correctly/

Reply:
agree we should add another releasename here and expect 2 releases

Reply:
ah, understood! I added another test for that.